db/migrate_cards/20160108030115_import_general_overview_related_updated.rb

```diff
@@ -0,0 +1,8 @@
+# -*- encoding : utf-8 -*-
+
+class ImportGeneralOverviewRelatedUpdated < Card::Migration
+  def up
+    import_json "general_overview_related_updated.json"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.